.github/workflows/tag-nightly.yml

```diff
@@ -0,0 +1,16 @@
+name: Tag iroha 2 as nightly
+
+on:
+  workflow_dispatch:
+
+jobs:
+  tag-nightly:
+    runs-on: [self-hosted, Linux]
+    steps:
+      - name: Tag
+        env:
+          origin: ${{ secrets.DOCKERHUB_ORG }}/iroha:iroha2-dev
+          nightly_prefix: ${{ secrets.DOCKERHUB_ORG }}/iroha:iroha2-nightly-
+        run: |
+          docker pull $origin
+          docker tag $origin "$nightly_prefix-$(date '+%y-%m-%d')"
```

Comment:
Also Rust adds date and commit for nightly https://rust-lang.github.io/rustup/concepts/channels.html#working-with-nightly-rust

I think we should also have a commit hash